src/HSE/All.hs

```diff
@@ -242,7 +255,11 @@ parseModuleEx flags file str = timedIO "Parse" file $ do
                     | otherwise -> readFileUTF8' file
         str <- return $ fromMaybe str $ stripPrefix "\65279" str -- remove the BOM if it exists, see #130
         ppstr <- runCpp (cppFlags flags) file str
-        dynFlags <- parsePragmasIntoDynFlags baseDynFlags file ppstr
+        -- Pull out langexts passed by config or command line to
+        -- explicitly enable/disable
+        -- (see https://github.com/ndmitchell/hlint/issues/681).
```

Comment:
Again, don't see this comment as valuable since the code is pretty clear

Reply:
Ok.